prettier.config.js

```diff
@@ -1,8 +1,19 @@
 module.exports = {
```

Comment:
I would prefer a `.prettierrc.json` over a js file for autocomplete and tooltips and since this type of config file that starts with a `.` typically isn't visible since and is barely touched.

Reply:
File is removed due to global config